docs/sources/reference/api/docker_remote_api.md

```diff
@@ -68,6 +68,13 @@ are now returned as boolean instead of as an int.
 In addition, the end point now returns the new boolean fields
 `CpuCfsPeriod`, `CpuCfsQuota`, and `OomKillDisable`.
 
+**New!**
+
+You can now specify a volume plugin in `/v1.19/containers/create`, for example
+`"HostConfig": {"Binds": ["flocker/name:/data"]}` where `flocker` is the name
```

Comment:
This should now read `"VolumeDriver": "flocker", HostConfig: {"Binds": ["name:/data"]}`, I think. @calavera can you double check this and then make the change?